ducktape/cluster/linux_remoteaccount.py

```diff
@@ -12,15 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
 from ducktape.cluster.cluster_spec import LINUX
 from ducktape.cluster.remoteaccount import RemoteAccount
 
 
 class LinuxRemoteAccount(RemoteAccount):
 
-    def __init__(self, ssh_config, externally_routable_ip=None, logger=None):
-        super(LinuxRemoteAccount, self).__init__(ssh_config, externally_routable_ip=externally_routable_ip,
-                                                 logger=logger)
+    def __init__(self, *args, **kwargs):
```

Comment:
there used to be a `logger` arg, but I don't see it being passed in in the `make_remote_account` method. Was anyone using it?

Reply:
Nah, I was mostly making sure that you don't get any unintentional consequences of passing all the args from the method above, but I don't see a real issue anywhere